docs/developer/index.rst

```diff
@@ -13,14 +13,16 @@ Before you submit a pull request to Bodhi, please ensure that it meets these cri
 * All tests must pass.
 * New code must have 100% test coverage. This one is particularly important, as we don't want to
   deploy any broken code into production. At the end of ``btest`` run, you can see your code coverage.
+  If you are not using Vagrant environment you can check the code coverage by running
+  ``diff-cover coverage.xml --compare-branch=origin/develop --fail-under=100``.
 * New functions, methods, and classes must have docblocks that explain what the code block is, and
   describing any parameters it accepts and what it returns (if anything). You can use the
-  ``pydocstyle`` utility to automatically check your code for this. You can also run ``bci pydocstyle``
+  ``pydocstyle`` utility to automatically check your code for this. You can also run ``bteststyle``
```

Comment:
Actually, running `bteststyle` will run flake8, pydocstyle and mypy, so I think the document is a bit misleading here. The in-vagrant command to only run pydocstyle is `pydocstyle bodhi`.